_posts/2020-06-24-cockpit-222.md

```diff
@@ -0,0 +1,39 @@
+---
+title: Cockpit 222
+author: pitti
+date: 2020-06-24
+tags: cockpit linux
+slug: cockpit-222
+category: release
+summary: Cockpit with Logs filtering improvements and web server hardening
+comments: 'true'
+---
+
+Cockpit is the [modern Linux admin interface](https://cockpit-project.org/). We release regularly. Here are the release notes from version 222.
+
+### Logs: More flexible text filters
+
+The text filter gained new `since:` and `boot:` quantifiers which restrict the logs to a given time span or the number of a previous boot; for example, `since:-12hours` or `boot:-2`. The "Time" dropdown controls these fields, and continues to have the same standard options as before, but they can now be customized easily.
+
+The `service:`*unitname* filter now accepts a unit name without the `.service` suffix. This mirrors the [systemctl](https://www.freedesktop.org/software/systemd/man/systemctl.html) command line behaviour.
+
+![Logs filtering by time and boot](/images/logs-filter-since-boot.png)
+
+### Services, Dashboard: Hide some buttons when access is limited
+
+Similarly to previous releases, the Services and Dashboard pages will now hide buttons for privileged actions when the logged in user doesn't have administrative access.
```

Comment:
to "the" previous? (I should not really be doing grammar review, I cannot even properly speak Slovak :D)

Reply:
My builtin English grammar-o-meter says that it's ok -- "the" is definite, without "the" it's indefinite, and I don't want to pin down which past releases in particular.